<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="5">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Hash Input:   (sys encrypted login hash, extracted from encrypted office/pdf/archive, ... ...)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QTextEdit" name="inTextEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="switchForceRBtn">
        <property name="text">
         <string>Btn</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="3">
       <widget class="QPushButton" name="chkBtn">
        <property name="text">
         <string>force crack</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="5">
       <widget class="QTextEdit" name="resultEdit">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>min len:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="5">
       <widget class="QLabel" name="chkLabel">
        <property name="text">
         <string>waiting for check ... ...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QLineEdit" name="inPwdEdit"/>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Corresponding Password for test:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QComboBox" name="typeComboBox"/>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="deviceComboBox"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="minLenEdit"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>max len:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="maxLenEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
